### PR TITLE
migrate to latest pkg.

### PR DESCRIPTION
**Pull request by n3wscott**
## Proposed Changes

  * duckv1alpha1 status was deleted. migrate off.
  * Some behavior change with MarkTrue on conditions, fixed.
  * No real code change

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go [hunk 1]
```diff
@@ -119,12 +116,19 @@ func (s *AwsSqsSourceStatus) InitializeConditions() {
 
 // MarkSink sets the condition that the source has a sink configured.
 func (s *AwsSqsSourceStatus) MarkSink(uri string) {
-	s.SinkURI = uri
 	if len(uri) > 0 {
-		condSet.Manage(s).MarkTrue(AwsSqsSourceConditionSinkProvided)
+		if u, err := apis.ParseURL(uri); err != nil {
+			s.SinkURI = nil
+			condSet.Manage(s).MarkUnknown(AwsSqsSourceConditionSinkProvided,
+				"SinkEmpty", "Sink resolving resulted in an error. %s", err.Error())
```
Comment (grantr) [inline awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go]: This is a bug fix right? Does it change the behavior when a referenced sink has an invalid addressable uri? If so add it to release notes.

File: pkg/controller/sdk/status_accessor_test.go [hunk 2]
```diff
@@ -1,114 +0,0 @@
-/*
```
Comment (grantr) [inline pkg/controller/sdk/status_accessor_test.go]: Why did these tests get deleted but not the code under test? Is that code unused? Are the tests invalid?

Comment (n3wscott) [reply pkg/controller/sdk/status_accessor_test.go]: the entire sdk needs to get deleted and I did not want to migrate the tests off the v1alpha1 Status

File: awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go (hunk 1 above)
Comment (n3wscott) [reply awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go]: it should be exactly the same.

Comment (n3wscott) [reply awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go]: and if it was important there would be a test. Tests passed.

Comment (grantr) [reply awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go]: Users don't care about tests or not, they care about what changed. We have lots of untested behavior. But if you're sure it's the same, no need for a release note.

File: pkg/controller/sdk/status_accessor_test.go (hunk 2 above)
Comment (grantr) [reply pkg/controller/sdk/status_accessor_test.go]: IMO if the code is still used, it should have tests.

File: awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go (hunk 1 above)
Comment (n3wscott) [reply awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types.go]: I am going to delete the sdk. NOW.

File: pkg/controller/sdk/status_accessor_test.go (hunk 2 above)
Comment (n3wscott) [reply pkg/controller/sdk/status_accessor_test.go]: https://github.com/knative/eventing-contrib/issues/947

**Pull request merged**